test/tests/functional/pbs_cray_suspend_resume.py

```diff
@@ -0,0 +1,91 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2016 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# The PBS Pro software is licensed under the terms of the GNU Affero General
+# Public License agreement ("AGPL"), except where a separate commercial license
+# agreement for PBS Pro version 14 or later has been executed in writing with
+# Altair.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software - under
+# a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+import time
+from tests.functional import *
+
+
+@tags('cray')
+class TestSuspendResumeOnCray(TestFunctional):
+    """
+    Test special cases where suspend/resume functionality differs on cray
+    as compared to other platforms.
+    """
+    def setUp(self):
+        if self.du.get_platform() is not "cray":
+            self.skipTest("Test suite only meant to run on a Cray")
```

Comment:
You might also want to skip if suspend-resume is not enabled on the Cray.

Reply:
@vccardenas Thanks for reviewing!
I can skip the test but just checking a parameter in a config file to decide that suspend/resume on the system is enabled would not be enough because for this configuration to take effect even ALPS needs to be restarted.
Secondly, there might be cases where ALPS config file may be present in different path in different CLEs.
For the reasons stated above I'd rather add a comment in the PTL script stating that the script expects suspend/resume being enabled on the system.

Reply:
Yes, adding a comment seems like a good idea.